### ISSUE TITLE
trace/performance is not being collected/pushed

### ISSUE THREAD

**Issue by douglasjam** [getsentry/sentry-symfony#442]
I'd like to enable traces with my Symfony application, but I tried and dug into the bundle, but couldn't manage to make it work, simply nothing appears on the website.

Issues/exceptions are being collected properly but traces not.

Do I need to do some extra configuration? How can I debug why it doesn't work properly? What is missing? I added breakpoints in the `isTracingEnabled` and `startTransaction` but doesn't go there.

### Environment
symfony 5.2
sentry SaaS
php 8.02
sentry/sentry-symfony 4.0

### Steps to Reproduce

I have the following configuration

packages/sentry.yaml
```
sentry:
    dsn: '%env(SENTRY_DSN)%'
    register_error_listener: true
    options:
        traces_sample_rate: 1.0
```

services.yaml
```
    Sentry\Monolog\Handler:
        public: true
        arguments:
            $level: !php/const Monolog\Logger::WARNING
```

**Comment by Jean85** [comment]
Unfortunately that's something that is still being worked on. You can look at our progress in #423 and subsequent #426 and #430.
The DB tracing is proving to be more difficult that expected due to Doctrine libraries being at a crossroad between 2.x and 3.x.

**Comment by ste93cry** [comment]
I'm closing this because the performance tracing has been released with version [`4.1.0`](https://github.com/getsentry/sentry-symfony/releases/tag/4.1.0)
